### ISSUE TITLE
Decreased circulating sulfate level

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9977]
**Preferred term label:**
Decreased circulating sulfate level

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://omim.org/entry/620372

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6000854
- label: Decreased circulating sulfate concentration
- parents: [Abnormal blood ion concentration (HP:0003111)](https://hpo.jax.org/app/browse/term/HP:0003111)
- pmid: PMID:36719378
- def: The concentration of sulfate in the blood circulation is below the lower limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9977
- synonyms:
